astroquery/gaia/core.py

```diff
@@ -772,6 +830,15 @@ def __getCoordInput(self, value, msg):
 
     @staticmethod
     def correct_table_units(table):
+        """Correct format in the units of the columns
+        TAP & TAP+
+
+        Parameters
+        ----------
+        table :Table, mandatory
```

Comment:
sphinx needs spaces on both sides of the `:` when listing parameters (this won't cause a warning/error, just won't render properly)
```suggestion
        table : `~astropy.table.Table`, mandatory
```